app.py

```diff
@@ -52,6 +54,9 @@
     corr_comparison_score.subscribe(weirdness_score)
     weirdness_score.subscribe(print)
 
+    if "FLT_INFLUX_HOST" in os.environ:
+        influxdb_storage = flatliners.InfluxdbStorage()
```

Comment:
on the next refactoring, you might pass the value `FLT_INFLUX_HOST` to `InfluxdbStorage` to reduce the coupling